auto-cpufreq-installer

```diff
@@ -192,6 +192,8 @@ elif [ -f /etc/os-release ];then
 else # In case /etc/os-release doesn't exist
 	manual_install
 fi
+# to clone the repo
+mkdir /opt/auto-cpufreq/source
 }
```

Comment:
Let's only run this if `--update` is run, as otherwise when If `sudo ./auto-cpufreq-installer --install` is run, and `/opt/auto-cpufreq/source` was created before following error will pop up:

`mkdir: cannot create directory ‘/opt/auto-cpufreq/source’: File exists`

Reply:
Let's move 

```
# to clone the repo
mkdir /opt/auto-cpufreq/source
```

after

```
elif update:
```

in `bin/auto-cpufreq` file, as this way `/opt/auto-cpufreq/source` won't be created until you run `auto-cpufreq --update`

You can even use do it using:

```
if not os.path.exists(custom_dir):
    os.mkdir(custom_dir)
```

Since custom_dir variable already exists.

Reply:
But now I see you're creating this as part line 254 of `bin/auto-cpufreq` file which makes these 2 lines obsolete?

So let's remove them

```
 to clone the repo
mkdir /opt/auto-cpufreq/source
```